cfgov/unprocessed/css/search.less

```diff
@@ -0,0 +1,110 @@
+.search_wrapper {
```

Comment:
The CSS in this file is pulled from the eregs app's `search.less` with some omissions. In a future PR I'll update eregs so both apps import this shared code, but I want to limit the external impacts of this initial PR to make it easier to review.